data/com/mojang/blaze3d/vertex/VertexMultiConsumer.mapping

```diff
@@ -40,3 +47,8 @@ CLASS com/mojang/blaze3d/vertex/VertexMultiConsumer
 			ARG 12 normalX
 			ARG 13 normalY
 			ARG 14 normalZ
+	CLASS Multiple
+		METHOD <init> ([Lcom/mojang/blaze3d/vertex/VertexConsumer;)V
+			ARG 1 delegates
+		METHOD forEach (Ljava/util/function/Consumer;)V
+			ARG 1 consumerConsumer
```

Comment:
I hate this, but I have no idea what else to name it.

Reply:
:shipit: